node-runner-cli/setup/SystemDSetup.py

```diff
@@ -462,8 +463,6 @@ def install_systemd_service(settings: SystemDConfig, args):
 
     @staticmethod
     def chown_files(settings):
-        run_shell_command(['sudo', 'chown', 'radixdlt:radixdlt',
```

Comment:
Same as the above comment
https://github.com/radixdlt/babylon-nodecli/pull/100/files#r1388101674

Reply:
added the command back with fail_on_error=False